activemodel/test/cases/errors_test.rb

```diff
@@ -305,4 +306,5 @@ def test_has_no_key
     person.errors.expects(:generate_message).with(:name, :blank, { message: 'custom' })
     person.errors.add_on_blank :name, message: 'custom'
   end
+
```

Comment:
This is diff noise, it would be nice if this was reverted to minimize the noise.